meiki_ui/src/components/app/userForm/LoginUserForm.svelte

```diff
@@ -15,10 +14,15 @@
     async function onClick() {
         try {
             await login(userForm.getUsername(), userForm.getPassword())
-
             dispatch("userLoggedIn")
         } catch (err) {
-            error = "An error has occured while logging in, try again"
+            if (err instanceof StatusNotOkError) {
+                error = await err.message
+                return
+            }
+
+            error =
+                "An error has occured while creating the account, please try again later"
```

Comment:
```suggestion
                "An error has occured while creating logging in, please try again later"
```

Reply:
Yep

Reply:
Can you change the suggestion to remove "creating"?

Reply:
sorry, done